java-checks/src/main/java/org/sonar/java/symexecengine/DataFlowVisitor.java

```diff
@@ -62,6 +80,9 @@ public void visitVariable(VariableTree tree) {
     if (isSymbolRelevant(tree.symbol())) {
       executionState.defineSymbol(tree.symbol());
       executionState.createValueForSymbol(tree.symbol(), tree);
+      if (tree.initializer() != null) {
+        check.onAssignment(executionState, tree, tree.symbol(), tree.initializer());
```

Comment:
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Method 'SymbolicExecutionCheck.onAssignment(...)' is deprecated.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)